src/openeo_gfmap/manager/job_splitters.py

```diff
@@ -60,17 +60,23 @@ def split_job_s2grid(
     if polygons.crs is None:
         raise ValueError("The GeoDataFrame must contain a CRS")
 
-    polygons = polygons.to_crs(epsg=4326)
-    if polygons.geometry.geom_type[0] != "Point":
-        polygons["geometry"] = polygons.geometry.centroid
+    original_crs = polygons.crs
+
+    # Transform to web mercator, to calculate the centroid
+    polygons = polygons.to_crs(epsg=3857)
+
+    polygons["centroid"] = polygons.geometry.centroid
 
     # Dataset containing all the S2 tiles, find the nearest S2 tile for each point
     s2_grid = load_s2_grid()
+    s2_grid = s2_grid.to_crs(epsg=3857)
```

Comment:
is this not a slow step for the entire s2 grid?

Reply:
@VincentVerelst Maybe if this is a speed issue you could upload a second s2 grid in the artifactory with the CRS reprojected, and then change the `load_s2_grid()` function in gfmap to accept a parameter `web_mercator: bool = False`

Reply:
May be worth the effort indeed.

Reply:
Thanks for the suggestion, I'll already update that.

Do have to note that the largest bottleneck is typically not the CRS conversion of the s2 grid, but rather the CRS conversion of the dataframe to split itself.

For example, for `2018_BEL_LPIS-Flanders_POLY_110` the old job_splitter takes 10s to run, while the new one takes 20s.